client/asset/dcr/simnet_test.go

```diff
@@ -340,24 +353,30 @@ func runTest(t *testing.T, splitTx bool) {
 		t.Fatalf("redemption error: %v", err)
 	}
 
+	betaSPV := rig.beta().wallet.SpvMode()
+
 	// Find the redemption
 	swapReceipt := receipts[0]
-	waitNetwork()
-	ctx, cancel := context.WithDeadline(tCtx, time.Now().Add(time.Second*5))
-	defer cancel()
-	_, checkKey, err := rig.beta().FindRedemption(ctx, swapReceipt.Coin().ID(), nil)
-	if err != nil {
-		t.Fatalf("error finding unconfirmed redemption: %v", err)
-	}
-	if !bytes.Equal(checkKey, secretKey1) {
-		t.Fatalf("findRedemption (unconfirmed) key mismatch. %x != %x", checkKey, secretKey1)
+	// The mempool find redemption request does not work in SPV mode.
+	if !betaSPV {
+		waitNetwork()
+		ctx, cancel := context.WithDeadline(tCtx, time.Now().Add(time.Second*5))
+		defer cancel()
+		_, checkKey, err := rig.beta().FindRedemption(ctx, swapReceipt.Coin().ID(), nil)
```

Comment:
Maybe just a `if dcr.wallet.SpvMode() { return }` at the start of `dcr.findRedemptionsInMempool`?

Reply:
Yeah, probably reasonable. Just wanted to make these test work for now, and even with said change to `findRedemptionsInMempool` we'd have to skip this `FindRedemption`.  I think?

Reply:
Yeah, otherwise, the mempool search will be skipped but `FindRedemption` will block waiting for new blocks to check until the context cancels and an error is returned to the caller.